app/src/main/java/ca/chronofit/chrono/util/components/Chronometer.kt

```diff
@@ -38,6 +38,8 @@ class Chronometer @JvmOverloads constructor(
 
     private var lastSecond = -1
 
+    private var notificationSetting = true
```

Comment:
notificationEnabled

Reply:
Instead of having this default to true, should we have a constructor that takes in notificationEnabled as a param? That way we can establish notificationEnabled's state without having to call the setter after initilization.

Reply:
Fixed this